teams/vDevPreview/MicrosoftTeams.schema.json

```diff
@@ -2894,24 +2944,119 @@
             },
             "additionalProperties": false
         },
-        "declarativeCopilotRef": {
+        "declarativeAgentRef": {
             "type": "object",
             "properties": {
               "id": {
                 "type": "string",
-                "description": "A unique identifier for this declarative copilot element."
+                "description": "A unique identifier for this declarative agent element."
               },
               "file": {
                 "$ref": "#/definitions/relativePath",
-                "description": "Relative file path to this declarative copilot element file in the application package."
+                "description": "Relative file path to this declarative agent element file in the application package."
               }
             },
-            "description": "A reference to a declarative copilot element. The element's definition is in a separate file.",
+            "description": "A reference to a declarative agent element. The element's definition is in a separate file.",
             "required": [
               "id",
               "file"
             ],
             "additionalProperties": false
+        },
+        "elementReference": {
+            "type": "object",
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "enum": [
+                        "bots",
+                        "staticTabs",
+                        "composeExtensions",
+                        "configurableTabs"
+                    ]
+                },
+                "id": {
```

Comment:
need to add id property to configurableTabs and composeExtensions

Reply:
Added